tests/travis/run_test.sh

```diff
@@ -59,7 +59,7 @@ if [ ${TASK} == "python_test" ]; then
         python -m pip install ./python-package/dist/xgboost-*-py3-none-${TAG}.whl
         conda --version
         python --version
-        python -m pytest -v -s -rxXs --durations=0 --fulltrace tests/python --cov=python-package/xgboost || exit -1
+        python -m pytest -v -s -rxXs --durations=0 --fulltrace tests/python/test_tracker.py --cov=python-package/xgboost || exit -1
```

Comment:
We should plan to restore the other unit tests for ARM64.

Reply:
Reverted the changes.